src/main/resources/public/jsx/Dataset.jsx

```diff
@@ -162,7 +163,8 @@ class Dataset extends React.Component {
             response.text().then(text => this.setState({ success : false, message: text }))
         })
     }
-    toggleCheckbox(label){
+    toggleCheckbox(event){
+        let label = event.target.value
```

Comment:
Since you don't modify its content, you should use `const` instead of `let`

Reply:
Fixed